src/RawPOSIX/src/safeposix/dispatcher.rs

```diff
@@ -501,6 +502,20 @@ pub fn lind_syscall_api(
             interface::cagetable_getref(cageid)
                 .fstat_syscall(fd, buf)
         }
+
+        UNLINKAT_SYSCALL => {
+            let fd = arg1 as i32;
+            
+            let pathname = match interface::types::get_cstr(start_address + arg2) {
```

Comment:
this needs to be updated to match the new dispatcher format now that the vmmap is merged